src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -567,6 +568,26 @@ def test_when_relationship_is_None_then_exception_raised(
         assert not mock_manager_interface.mock.getWithRelationships.called
 
 
+class DyingPagerInterface(EntityReferencePagerInterface):
+    """
+    Throwaway pager interface def, so we can create a temporary
+    interface intended to fall out of scope.
+    Any interface would do, but this avoids ugly imports.
```

Comment:
[question] Is this true, as it goes through C++ then it needs to inherit from `EntityReferencePagerInterface` right?

Reply:
I guess it's meant to read as "any EntityReferencePagerInterface would do"

Reply:
Just feels like the comment isn't really adding any clarity to the test setup/purpose?

Reply:
Dodged by simply removing that sentence as part of cf155b54